.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ parameters:
   # If you need to force build all of the packages, set default: true
   force-build-all:
     type: boolean
-    default: false
+    default: true
```

Comment:
I think we ought to set this back to false before merging

Reply:
If I do that, I won't see any tests or coverage reports being run, because I have no code changes, there is no build of these components.

Reply:
Ok, we'll need a second PR to reset it to false once we've shown it working in master then!